conviva/src/utils/Utils.ts

```diff
@@ -220,18 +223,19 @@ export function calculateBufferLength(player: ChromelessPlayer): number {
     return bufferLength * 1000;
 }
 
-export function flattenAndStringifyObject(obj: any): { [key: string]: string } {
-    const result: Record<string, string> = {};
-    Object.keys(obj).forEach((key) => {
-        try {
-            if (typeof obj[key] === 'object' && obj[key] !== null) {
-                result[key] = JSON.stringify(obj[key]);
-            } else {
-                result[key] = obj[key].toString();
-            }
-        } catch (ignore) {
-            // Failed to stringify value.
+export function flattenErrorObject(error?: THEOplayerError): { [key: string]: string } {
+    const errorDetails: Record<string, any> = {
```

Comment:
Now the value is of type `any` while previously we would always pass it as a `string`. Is this fine on conviva end or should we stringify the values to be safe?

Reply:
@Jeroen-Veltmans I updated the typings: we always pass a `{ [key: string]: string }` object to Conviva after all `undefined` or `null` values are removed from the object.